app/controllers/api/v2/config_reports_controller.rb

```diff
@@ -37,8 +37,8 @@ def show
       param_group :config_report, :as => :create
 
       def create
-        @config_report = ConfigReport.import(params[:config_report], detected_proxy.try(:id))
-        process_response @config_report.errors.empty?
+        task = ForemanTasks.async_task Actions::Foreman::Report::Import, params, ConfigReport, detected_proxy.try(:id)
```

Comment:
Similarly as with classes import, we should not push the whole params to the action, just the data we're interested in